content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,291 @@
+instead of services. If a policy applies to a workload that is used by multiple different
+services, the same policy will affect the traffic to all the different services.
+
+You can simply leave the `selector` empty to apply the policy to all workloads
+in a namespace. The following policy applies to all workloads in the namespace `bar`:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: policy
+ namespace: bar
+spec:
+ rules:
+ # omitted
+{{< /text >}}
+
+### Root namespace
+
+A policy in the root namespace applies to all workloads in the mesh in every namespaces.
+The root namespace is configurable in the [`MeshConfig`](/docs/reference/config/istio.mesh.v1alpha1/#MeshConfig)
+and has the default value of `istio-system`.
+
+For example, you installed Istio in `istio-system` namespace and deployed workloads in
+`default` and `bookinfo` namespace. The root namespace is configured to `istio-config`.
+The following policy will apply to workloads in every namespace including `default`,
+`bookinfo` and also the `istio-system`.
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: policy
+ namespace: istio-config
+spec:
+ rules:
+ # omitted
+{{< /text >}}
+
+### Ingress/Egress Gateway support
+
+The `v1beta1` authorization policy also supports to be applied on ingress/egress
+gateway to enforce access control on traffic entering/leaving the mesh, you just
+need to change the `selector` to make the policy to select the ingress/egress workload.
+
+The following policy applies to the workload with label `app: istio-ingressgateway` in
+`istio-system` namespace:
+
+{{< text yaml >}}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+ name: ingress
+ namespace: istio-system
+spec:
+ selector:
+   matchLabels:
+     app: istio-ingressgateway
+ rules:
+ # omitted
+{{< /text >}}
+
+### Comparision
+
+The following table highlights the key differences between the old `v1alpha1` RBAC policies and
+the new `v1beta1` authorization policy.
+
+| Feature | `v1alpha1` RBAC policy | `v1beta1` Authorization Policy |
+|---------|------------------------|--------------------------------|
+| API stability | `alpha`: **No** backward compatible | `beta`: backward compatible **guaranteed** |
+| Number of CRDs | Three: `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding` | Only One: `AuthorizationPolicy` |
+| Mechanism to specify where to apply the policy | Specify by **service** | Specify by **workload** |
+| Deny-by-default behavior | Enabled **explicitly** by configuring `ClusterRbacConfig` | Enabled **implicitly** with `AuthorizationPolicy` |
+| Ingress/Egress gateway support | Not supported | Supported |
+| The `"*"` value in policy | Match all contents (empty and non-empty) | Match non-empty contents only |
+| The `destination.user` constraint | Supported | Not supported |
```

Comment:
Is "destination.user" the only attribute not supported in beta policy? I do not think we need to mention it here.

Reply:
I think it's the only attribute. some user may be using it so I feel it's better for us to call out this change explicitly.